### ISSUE TITLE
Show dbt model freshness in Lightdash UI

### ISSUE THREAD

**Issue by owlas** [lightdash/lightdash#621]
It would be useful to know when a dbt model was last updated. This context would help users understand if an explore is out of date.

**Comment by owlas** [comment]
@djayatillake I think this also relates to #263 as well as having broader implications

**Comment by owlas** [comment]
As well as making this available via the API and via the lightdash CLI, there are two other places we could pull data:

- dbt cloud metadata api
- metaplane.dev

**Comment by TuringLovesDeathMetal** [comment]
If you have joined tables, would you be able to see the freshness of those joined tables separately? 

Like, beside the joined model or something in the sidebar? 

I'd love to know a bit more about _**where**_ people are expecting to use this as well. Is it while they're in the explore view? is it when they're looking at a chart?

**Comment by david-brandalley** [comment]
Yes I can't see why not, especially with Metaplane. It's a case of requesting status of all the tables in from our API and then being able to split the output and hang it on each table or column.

I think you should see it at the top of a pre-built chart that's in a dashboard or otherwise. Some kind of warning message to show that there is a potential problem with the data or a nice green tick or something saying all tests have passed.

When you're building a chart in an explore, you could probably show table level info on the tables above the fields, then field level info on the fields. Probably should be expandable to conserve space in both cases here.

**Comment by djayatillake** [comment]
Yes I can't see why not, especially with Metaplane. It's a case of requesting status of all the tables in from our API and then being able to split the output and hang it on each table or column.

I think you should see it at the top of a pre-built chart that's in a dashboard or otherwise. Some kind of warning message to show that there is a potential problem with the data or a nice green tick or something saying all tests have passed.

When you're building a chart in an explore, you could probably show table level info on the tables above the fields, then field level info on the fields. Probably should be expandable to conserve space in both cases here.

**Comment by 12ian34** [comment]
Was requested by a prospect today

**Comment by vijay-gamurs** [comment]
Can this be prioritized?

**Comment by jpetey75** [comment]
@oli-rmsy is looking at maybe a way to embed freshness in dashboard markdown tiles using this: https://docs.getdbt.com/docs/collaborate/data-tile which might be a good quick way to do this

2 customer mentions of this feature in slack: 

1. https://lightdash.slack.com/archives/C03HJ10UYQK/p1660575022034509?thread_ts=1659431851.757629&cid=C03HJ10UYQK
2. https://lightdash.slack.com/archives/C064W44EL4S/p1712931651845269?thread_ts=1712930122.696189&cid=C064W44EL4S

2 mentions (that i could find) from demo calls:

1. https://lightdash.slack.com/archives/C02UE13VBNC/p1686344263371709?thread_ts=1686344175.901719&cid=C02UE13VBNC
2. https://lightdash.slack.com/archives/C02UE13VBNC/p1676574545460939
